app/assets/javascripts/spree/frontend/solidus_acima.js

```diff
@@ -1,2 +1,8 @@
 // Placeholder manifest file.
-// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
\ No newline at end of file
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
+
+const acima = new Acima.Client({
+  merchantId: "loca-0af305b8-23d6-4c60-9607-50a490872b0a",
```

Comment:
despite being define on lib/generators/solidus_acima/install/templates/initializer.rb these are static here, what is the reason? 🤔

Reply:
ah, sorry, forgot to remove it